### PR TITLE
Add docker-compose and config files

### PR DESCRIPTION
**Pull request by OlegKDev**

### CODE REVIEW

File: nginx/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+server {
```
Comment (Denis-26) [inline nginx/nginx.conf:1]: other one

```
server {

    listen 443 ssl;
    server_name push.all-easy.ru;

    root /usr/share/nginx/html/push.all-easy.ru;
    index index.html index.htm;

    location /v1/api/ {
        proxy_set_header X-Real-IP $remote_addr;
        proxy_pass http://push-service:8080;
    }

    ssl on;
    ssl_certificate <path_to_fullchain>;
    ssl_certificate_key <path_to_privkey>;

    ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;

}
```

File: nginx/nginx-80.conf [hunk 2]
```diff
@@ -0,0 +1,13 @@
+server {
```
Comment (Denis-26) [inline nginx/nginx-80.conf]: this can be removed

File: nginx/grafana.all-easy.ru.conf [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    location /api/live/ {
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection $connection_upgrade;
+        proxy_set_header Host $http_host;
+        proxy_pass http://grafana;
+    }
+
+    ssl on;
+    ssl_certificate /etc/nginx/ssl/grafana.all-easy.ru/fullchain1.pem;
```
Comment (Denis-26) [inline nginx/grafana.all-easy.ru.conf]: replace with 

```
ssl_certificate <path_to_fullchain>;
ssl_certificate_key <path_to_privkey>;
```

File: push/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,7 @@
+FROM openjdk:17-jdk-slim
```
Comment (Denis-26) [inline push/Dockerfile]: do not need now, cause we build Dockerfile on the flight

Comment (Denis-26) [reply push/Dockerfile]: so remove this file

File: push/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+version: "3"
```
Comment (Denis-26) [inline push/docker-compose.yml:1]: actual config is 

```
version: "3"
services:
  # APP*****************************************
  push-service:
    container_name: push-service
    image: dakotaio/push:${VERSION}
    networks:
      - postgres
      - metrics
    volumes:
      - ./log:/var/log
    expose:
      - 8080
    ports:
      - 8080:8080
    environment:
      - PROFILE=${PROFILE}
      - DB_USERNAME=${DB_USERNAME}
      - DB_PASSWORD=${DB_PASSWORD}
      - TG_HOOK_SECRET=${TG_HOOK_SECRET}
      - TG_HOOK_URL=${TG_HOOK_URL}
      - TG_TOKEN=${TG_TOKEN}
      - REDIS_PASSWORD=${REDIS_PASSWORD}
      - DB_NAME=${DB_NAME}
      - api_key=${API_KEY}
    restart: unless-stopped

networks:
  postgres:
    external:
      name: common
  metrics:
    external:
      name: metrics
```

File: nginx/grafana.all-easy.ru.conf (hunk 3 above)
Comment (OlegKDev) [reply nginx/grafana.all-easy.ru.conf]: Done

File: nginx/nginx-80.conf (hunk 2 above)
Comment (OlegKDev) [reply nginx/nginx-80.conf]: Done

File: push/Dockerfile (hunk 4 above)
Comment (OlegKDev) [reply push/Dockerfile]: Done

File: nginx/nginx.conf (hunk 1 above)
Comment (OlegKDev) [reply nginx/nginx.conf:1]: Done

File: push/docker-compose.yml (hunk 5 above)
Comment (OlegKDev) [reply push/docker-compose.yml:1]: Done

**Pull request merged**